database/migrations/2021_08_30_114640_create_products_table.php

```diff
@@ -19,8 +19,9 @@ public function up()
             $table->text('description');
             $table->float('price');
             $table->integer('quantity');
-            $table->foreignId('type_id')->constrained('product_types');
-            $table->foreignId('discount_id')->constrained('discounts');
+            $table->timestamps();
```

Comment:
we should avoid editing migrations, let's imaging following case:
1. another developer who also working with this app already applied this migration to his local database
2. we updated this migration
3. another developer fetches new functionality from repository, but he will not be able to apply those changes related to migration because his database already saved this migration name as applied

In case if we need to edit already applied migration -> we need create new migration.